src/routes/mix/mix.routes.ts

```diff
@@ -0,0 +1,68 @@
+import { Request, Response, Router } from 'express';
+import { handleError } from '@root/utils/handleError';
+import { IMixService, MixService } from '@root/domains/mix';
+import { DoneBody, NextDrinkBody } from './types';
+
+export const mixRoutes = () => {
+  const router = Router();
+  const mixService: IMixService = new MixService();
+
+  /**
+   * @swagger
+   * /mix/getnext:
+   *  post:
+   *    summary: Get next drink
+   *    description: "Get the next drink from the queue, belonging to the given machine id"
+   *    consumes: application/json
+   *    requestBody:
+   *      content:
+   *        $ref: '#/components/requestBodies/NextDrinkBody'
+   *    responses:
+   *      200:
+   *        $ref: '#/components/responses/MixDTO'
+   */
+  router.post(
+    '/getnext',
```

Comment:
Varför är detta ett post request? Som du har i din description så gör den bokstavligen `Get next drink`. Routes borde också inte ha "get" i namnet då http-metoden ska visa på det. Så om du vill mata in ett ID så hade det varit enklare och mer logiskt att routen heter `/queue/:mixerId/next` som är mer generiskt. Det är kanske inte det rätta namnet men dessa routes verkar ha mer korrelation till kön än faktiska maskinen då det kan vara vad som helt som vill hämta från kön

Reply:
för att post var det enda jag hade att jämföra med heh.. fixar det

Den bör vara för maskinen, datan den svarar med är nummer för flaskpositioner i maskinen.